doc/source/train/examples/lightning/lightning-llm-finetuning-7b.ipynb

```diff
@@ -0,0 +1,1265 @@
+{
```

Comment:
3B and 7B are that different? why do you need 2 scripts?

Reply:
Actually I want to only keep the 7B one. However the 7B model doesn't fit into a single GPU, thus I can't use huggingface pipeline in the inference section. Probably I can try to compress the model.
Do you have any thoughts on that?

Reply:
I feel like this is where you have to use Ray to do it.
Setup up a DDP worker group like what we do for the DeepSpeed inference example. Then run 1 worker per GPU.
You can then use FSDP to load the model and do forward pass under no_grad().
May not be the best solution, but a solution?

Reply:
Thanks Jun!  I'll try this approach. This is quite important since we have no support for distributed inference right now.

Also I found that huggingface `device_map="auto"` could work. Basically it offload params that can't fit in GPU to CPU memory. I'll probably use it for this example for now.